js/config.ts

```diff
@@ -177,7 +177,12 @@ const DEFAULT_CONFIG = {
   debug: false, // Adds features specific for debugging or model development, e.g. plates visibility toggle
   debugCrossSection: false,
   benchmark: false,
-  markCrossSectionFields: false
+  markCrossSectionFields: false,
+  // How often model web worker should send a message with fields data to the main UI thread.
+  fieldsUpdateInterval: 10,
+  // How often model web worker should send a message with cross-section data to the main UI thread.
+  crossSectionUpdateInterval: 5,
```

Comment:
In what unit?

Reply:
10 means that fields are updated every 10th message sent from worker to UI (and that depends on machine performance, but the target is 30 fps / model steps per second). So, the lower, the more often. I can add comments about that.